Library/Formula/vile.rb

```diff
@@ -0,0 +1,23 @@
+class Vile < Formula
+  desc "Vi Like Emacs"
+  homepage "http://invisible-island.net/vile/vile.html"
+  url "http://invisible-island.net/datafiles/release/vile.tar.gz"
```

Comment:
I looked a little more and found another link for the tarball with the version information.